test/battle/status1/sleep.c

```diff
@@ -22,6 +22,20 @@ SINGLE_BATTLE_TEST("Sleep prevents the battler from using a move")
     }
 }
 
+SINGLE_BATTLE_TEST("Sleep: Spore doesn't affect grass types")
```

Comment:
```suggestion
SINGLE_BATTLE_TEST("Sleep: Spore doesn't affect grass types (Gen 6+)")
```
Forgot to suggest this before